tests/conftest.py

```diff
@@ -382,7 +382,8 @@ def hg_server(hg_test_bundle, tmpdir):
     hg_url = "http://localhost:" + port
 
     repo_dir = tmpdir.mkdir("hg_server")
-    subprocess.run(["hg", "clone", hg_test_bundle, repo_dir], check=True)
+    subprocess.run(["hg", "clone", hg_test_bundle, repo_dir], 
+                   check=True, cwd="/")
```

Comment:
Without this I get an error on my local machine due to an extension I have installed. So setting to `/` will put the cwd outside of the mounted docker volume and avoid that problem, and since it's a clone the `cwd` doesn't really matter.